### PR TITLE
My Assignment 4 Submission

### PR DESCRIPTION
**Pull request by NagaarjunanL**

### CODE REVIEW

File: src/Assignment-4/assignment4.html [hunk 1]
```diff
@@ -6,12 +6,75 @@
...
+			<li><strong>Stuff</strong></li>
+		  </ul>
+		</nav>
+	  </header>
+  
+	  <section id="hero" >
+		  <div >
+		<h2>A Site About Some Dogs</h2>
+		<img src="images/Dog1.jpg" alt="hero dog"/>
+		<a href="http://corgis-in-space.tumblr.com/" target="_blank">What are Dogs</a>
```
Comment (rilwanhaq) [inline src/Assignment-4/assignment4.html:33]: can you correct the alignment of what are dogs?

File: src/Assignment-4/assignment4.html [hunk 2]
```diff
@@ -6,12 +6,75 @@
...
+  
+	  <footer>
+		<h4>Learn About Dogs Online</h4>
+		<ul>
+			<li><img src="images/FB icon.jpg"/></li>
+			<li><img src="images/twitter icon.jpg"/></li>
+			<li><img src="images/Insta icon.jpg"/></li>
+			<li><img src="images/LinkedIn icon.jpg"/></li>
+		</ul>
+		<a href="https://github.com/turingschool-examples/introductory-static-site" target="_blank" class="link">See this on GitHub</a>
```
Comment (rilwanhaq) [inline src/Assignment-4/assignment4.html:74]: this a tag should not look like button can you please change it?
Assignment-4 2

File: src/Assignment-4/assignment4.css [hunk 3]
```diff
@@ -5,3 +5,111 @@
...
+    font-family: 'Raleway', sans-serif;
+  }
+header {
+    background: #92E0E5;
+    color: white;
+    height: 80px;
+    padding: 0 25px;
+  }
+
+  header .logo {
```
Comment (rilwanhaq) [inline src/Assignment-4/assignment4.css:18]: border-radius:50%; 
can you do logo as circle

**Review by rilwanhaq — APPROVED** [review]
please correct those small alignment mistakes rest is fine .good work bro

**Comment by SuneetKumar19** [comment]
Fine!

**Pull request merged**